client/utils.js

```diff
@@ -37,16 +37,16 @@ const updateMessageEventCount = (messageId, event) => {
 };
 
 module.exports = {
-	generateMessageEvent: function ({ messageId, position, flag, variant }={}) {
+	generateMessageEvent: function ({ messageId, position, flag, trackingContext, variant }={}) {
```

Comment:
I would set a default value for trackingContext. 
```suggestion
	generateMessageEvent: function ({ messageId, position, flag, trackingContext = {}, variant }={}) {
```
or you could have an error when you do object.assign if that is not passed for other messages

Reply:
> `Object.assign()` does not throw on null or undefined sources.
> - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

I did test this on other messages and they worked fine. Can't hurt to put a default in though so I'll update it.

Reply:
Don't change it then. I thought it was throwing on `undefined`